tailcfg/tailcfg.go

```diff
@@ -1425,6 +1425,43 @@ type MapResponse struct {
 	// server. An initial nil is equivalent to new(ControlDialPlan).
 	// A subsequent streamed nil means no change.
 	ControlDialPlan *ControlDialPlan `json:",omitempty"`
+
+	// ClientVersion describes the latest client version that's available for
+	// download and whether the client is using it. A nil value means no change
+	// or nothing to report.
+	ClientVersion *ClientVersion `json:",omitempty"`
+}
+
+// ClientVersion is information about the latest client version that's available
+// for the client (and whether they're already running it).
+//
+// It does not include a URL to download the client, as that varies by platform.
+//
+// The main purpose of this field is for uncaching update checks in the Windows GUI
+// and
```

Comment:
Dangling `and` here.

Reply:
I was in the middle of moving this text down below to the fields and forgot to delete this. Deleted.